README.md

```diff
@@ -1 +1,198 @@
-<h3><b>Javascript Capstone Project</b></h3>
\ No newline at end of file
+<h3><b>Javascript Capstone Project</b></h3>
+ 
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+
+
+# 📖 [JavaScript Capstone] <a name="capstone-project"></a>
+
+> **A web application based on using external API's**
+
+## 🛠 Built With <a name="built-with">HTML, CSS, JavaScript  & API's</a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <ul>
+    <li><a href="">linters</a></li>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://www.w3schools.com/js/default.asp">JS</a></li>
+    <li><a href="https://github.com">GIT/Github</a></li>
+    <li><a href="https://www.themealdb.com/api.php">API</a></li>
+    <li><a href="https://www.notion.so/microverse/Involvement-API-869e60b5ad104603aa6db59e08150270">Involvement API</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+> **[Use JavaScript to make websites dynamic and build basic single page apps.]**
+> **[Write units tests for a JavaScript app]**
+> **[Send and receive data from an API]**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Loom Video](https://loom.com/share/e2f473991f3f4cc78aad4c24127a1ab5)
+- [Live Demo Link]()
```

Comment:
- [ ] Since you don't have a live link yet , kindly remove `Live Demo Link` from your readme file to make it looks professional 🙏